assets/js/src/common/confirm_alert.jsx

```diff
@@ -0,0 +1,60 @@
+import MailPoet from 'mailpoet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class ConfirmAlert extends React.Component {
+  static propTypes = {
+    title: PropTypes.string,
+    message: PropTypes.string.isRequired,
+    cancelLabel: PropTypes.string,
+    confirmLabel: PropTypes.string,
+    onConfirm: PropTypes.func.isRequired,
+  }
+
+  static defaultProps = {
+    title: MailPoet.I18n.t('confirmTitle'),
+    cancelLabel: MailPoet.I18n.t('cancelLabel'),
+    confirmLabel: MailPoet.I18n.t('confirmLabel'),
+  }
+
+  close = () => {
```

Comment:
Can we do this differently? We shouldn't touch `document` inside React. The next version will be angry at us. But Why don't we just hide the modal instead of removing it from the dom completely?

Reply:
Updated so modal element is created only once and hidden after unmounting component.